### DISCUSSION TITLE
去除tooltip hover上去的圆点

### DISCUSSION THREAD

**Discussion by tiansQAQ** [antvis/G2#5374] [Q&A] [answered]
- **G2 Version**:4.1.22
- **Platform**:Chrome
- **Mini Showcase(like screenshots)**:
- **CodePen Link**:

<img width="348" alt="image" src="https://github.com/antvis/G2/assets/46640919/b00da5b2-5995-4d82-a113-f43b2305c1e4">

tooltip hover上去怎么去除图片上面的圆点？

**Comment by ai-qing-hai** [comment, marked as answer]
@tiansQAQ    
chart.tooltip({
  showMarkers: false,
});
